consent2/bitfield.go

```diff
@@ -0,0 +1,60 @@
+package consent2
+
+import (
+	"fmt"
+)
+
+func parseBitField(data consentMetadata) (*consentBitField, error) {
+	vendorBitsRequired := data.MaxVendorID()
+
+	// BitFields start at bit 213. This means the last three bits of byte 26 are part of the bitfield.
```

Comment:
The comment is wrong. The bit field starts at bit 230 so it's the last 3 bits of byte 28 we care about. The code looks to be correct.